### PR TITLE
StatementNode insert/remove support for Block

### PR DESCRIPTION
**Pull request by cancerberoSgx**
support for insert/remove statements in a Block kind node

### CODE REVIEW

File: src/tests/compiler/statement/statementedNodeTests.ts [hunk 1]
```diff
@@ -290,6 +297,13 @@ describe(nameof(StatementedNode), () => {
         it("should remove statements at the end default clause", () => {
             doFirstChildTest<DefaultClause>(defaultClause, [1, 2], "switch (x) {\n    default:\n        x = 0;\n}", SyntaxKind.DefaultClause);
         });
+
+        it("should remove statements in a Block", () => {
+            const {sourceFile, firstChild} = getInfoFromTextWithSyntax<Block>  ('function():number{const a = 1, b = true;}', SyntaxKind.Block);
+            expect(!!firstChild.getStatementByKind(SyntaxKind.VariableStatement)).equals(true);
```
Comment (dsherret) [inline src/tests/compiler/statement/statementedNodeTests.ts]: Could you change this to use syntax like `expect(something).to.not.be.undefined`?

File: src/tests/compiler/statement/statementedNodeTests.ts [hunk 2]
```diff
@@ -214,6 +214,13 @@ describe(nameof(StatementedNode), () => {
             doFirstChildTest<DefaultClause>(defaultClause, 3, "newText;\nsecondText;", 2,
                 "switch (x) {\n    default:\n        x = 0;\n        y = 1;\n        break;\n        newText;\n        secondText;\n}", SyntaxKind.DefaultClause);
         });
+
+        it("should insert statements in a Block", () => {
+            const {sourceFile, firstChild} = getInfoFromTextWithSyntax<Block>  ('function():number{const a = 1, b = true;}', SyntaxKind.Block);
```
Comment (dsherret) [inline src/tests/compiler/statement/statementedNodeTests.ts]: Could you run `npm run lint` and then fix the linting errors?

File: src/tests/compiler/statement/statementedNodeTests.ts (hunk 1 above)
Comment (cancerberoSgx) [reply src/tests/compiler/statement/statementedNodeTests.ts]: done

File: src/tests/compiler/statement/statementedNodeTests.ts (hunk 2 above)
Comment (cancerberoSgx) [reply src/tests/compiler/statement/statementedNodeTests.ts]: npm run lint didn't reported any errors. Did you expected one ?

Comment (dsherret) [reply src/tests/compiler/statement/statementedNodeTests.ts]: Really? That's strange. Yeah it should for the single quotes and spacing. Should be double quotes and no spaces between `<Block>` and the open paren.

**Comment by dsherret** [comment]
Looks good. Thanks!

**Pull request merged**